### ISSUE TITLE
npm install fails with 'ERESOLVE unable to resolve dependency tree'

### ISSUE THREAD

**Issue by a-panickar** [wbhob/nest-middlewares#46]
Installing the middleware as per the documentation results in the below error.

Please also refer the attached logs.

```bash
npm install --save @nest-middlewares/helmet        

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: test-middleware@0.0.1
npm ERR! Found: @nestjs/common@8.2.0
npm ERR! node_modules/@nestjs/common
npm ERR!   @nestjs/common@"^8.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @nestjs/common@"^6.0.0" from @nest-middlewares/helmet@6.0.0
npm ERR! node_modules/@nest-middlewares/helmet
npm ERR!   @nest-middlewares/helmet@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/abhilash.panickar/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/abhilash.panickar/.npm/_logs/2021-11-13T20_44_04_211Z-debug.log
```
[2021-11-13T20_44_04_211Z-debug.log](https://github.com/wbhob/nest-middlewares/files/7532729/2021-11-13T20_44_04_211Z-debug.log)

[eresolve-report.txt](https://github.com/wbhob/nest-middlewares/files/7532731/eresolve-report.txt)

**Comment by micalevisk** [comment]
they didn't released a version of `@nest-middlewares/helmet` that supports nestjs v8. The only published nestjs v6 as peer dep.

**Comment by wbhob** [comment]
just published package version 7.0.0 with compatibility for nestjs 9.0
